TACC/Core-CMS-Custom
# task/WP-391: Entities feedback UTH

## Overview
Addresses a bug + improvement to the entities on the registration form
…

## Related

- [WP-391](https://jira.tacc.utexas.edu/browse/WP-391)
…

## Changes
- Made all `input` fields for NAIC and License codes type `text`, and simplified field audits to check for up-to 10 digits (addresses both UTH request and bug for values that are too large being written to db and failing, killing the entire entity)
- Added `small` element to the Medicaid choice for plan type w/ helper text per UTH
…

## Testing

1. Go to https://localhost:8000/register/request-to-submit
2. Fill out the form as normal up to the entities, then confirm:
     - For the license and NAIC code fields, only values between 0 (as well 0000000000) and 9999999999 are accepted
     - Some small text saying "(for state use only)" appears next to the "Medicaid" option under "Type of Plan" checklist (see image below in UI section for reference)
     - Repeat for any # of additional entity entries
3. Repeat step 2 and all sub-steps for the following:
     - https://localhost:8000/administration/list-registration-requests > any record > Actions column > Edit Record
     - https://localhost:8000/register/list-registration-requests > any record > Actions column > Renew Registration
     

## UI
<img width="537" alt="image" src="https://github.com/TACC/Core-CMS-Custom/assets/43251554/ba9c0c0f-e520-4bb1-aed6-be79d87a9ff3">

…
